examples/browser-to-browser/README.md

```diff
@@ -0,0 +1,63 @@
+# js-libp2p-webrtc Browser to Browser
+
+This example leverages the [vite bundler](https://vitejs.dev/) to compile and serve the libp2p code in the browser. You can use other bundlers such as Webpack, but we will not be covering them here.
+
+## Running the Relay Server
+
+For browsers to communicate, we first need to run the Go LibP2P relay server:
+
+```shell
+npm run go-relay
+```
+
+Alternatively, a NodeJS relay server is available:
+
+```shell
+npm run node-relay
+```
+
+Copy one of the multiaddresses in the output.
+
+## Running the Example
+
+In a separate console tab, install dependencies and start the Vite server:
+
+```shell
+npm i && npm run start
+```
+
+The browser window will automatically open.  Let's call this `Browser A`.
+Using the copied multiaddress from the Go or NodeJS relay server, paste it into the `Remote MultiAddress` input and click the `Connect` button.
+`Browser A` is now connected to the relay server.
+Copy the multiaddress located after the `Listening on` message.
```

Comment:
I only see "dialing" but I never see a listening on message. Are you able to get
this working? It fails for me on safari, firefox, and chromium (Brave)